pkg/apis/app/v2/kieapp_types.go

```diff
@@ -450,9 +450,12 @@ type CommonConfig struct {
 	DBPassword         string `json:"dbPassword,omitempty"`
 	AMQPassword        string `json:"amqPassword,omitempty"`
 	AMQClusterPassword string `json:"amqClusterPassword,omitempty"`
+	// Deprecated - Remove after 7.6.1 is also removed
```

Comment:
we will not make 7.6.1, can it be removed now?

Reply:
I'd imagine this needs to stay until 7.6.0 rolls off. We still support both 7.6 and 7.5.1 installs with the 7.7 operator.

Reply:
@spolti I don't want to remove it until version 7.7 is the last one supported by the operator to allow users still deploy v1 of the CRD.